source/sgp_mode/GenomeLibrary.h

```diff
@@ -280,6 +299,27 @@ sgpl::Program<Spec> CreateNotProgram(size_t length) {
   return program.Build(length);
 }
 
+sgpl::Program<Spec> CreateParasiteNotProgram(size_t length, int steal_count) {
+  ProgramBuilder program;
+  program.AddSteal(steal_count);
+  program.AddNot();
+  
+  return program.Build(length);
+}
+
+sgpl::Program<Spec> CreateMutualistNotProgram(size_t length) {
+  ProgramBuilder program;
+  program.AddNot();
+  program.AddDonate();
```

Comment:
Can you not just call Add("Donate")? The "AddDonate" method seems a bit useless at the moment